packages/services/f-braze-adapter/src/BrazeAdapter.js

```diff
@@ -0,0 +1,110 @@
+import GetConsumerRegistry from './services/BrazeConsumerRegistry';
+
+export default class BrazeAdapter {
+    /**
+     * Initializes the braze adapter and returns instance of the braze adapter class
```

Comment:
Should this read _... and returns *the* single instance of the braze adapter class_?

Reply:
Not actually a singleton.  Ignore me